test/net/sourceforge/kolmafia/request/CampgroundRequestTest.java

```diff
@@ -114,6 +114,22 @@ void updatesBookshelfLibramCasts() {
     }
   }
 
+  @Test
+  void doesNotCountFailedRests() {
```

Comment:
it would be nice to have a test confirming that `timesRested` increases outside this case, just to confirm the test is testing the right thing.